### PR TITLE
chore: basic balance trait impl

### PR DESCRIPTION
**Pull request by Rqnsom**
Full balance implementation is still to be done.

### CODE REVIEW

**Review by asmie — APPROVED** [review]
LGTM :+1:

File: src/balance.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+use move_core_types::{account_address::AccountAddress, vm_status::StatusCode};
+use move_vm_backend::balance::BalanceHandler;
+
+/// Balance adapter for provading basic access to balance cheques within the MoveVM.
```
[the remark this answers is not in the document]
Comment (Rqnsom) [reply src/balance.rs:4]: Good idea. Let's unify it under the doc in some of the future PRs when the whole concept is fully clear!

[the remark this answers is not in the document]
Comment (Rqnsom) [reply src/balance.rs]: Done

**Review by asmie — APPROVED** [review]
:+1:

**Pull request merged**